backend/molgenis-emx2-datamodels/src/test/java/org/molgenis/emx2/datamodels/TestLoaders.java

```diff
@@ -148,22 +115,30 @@ void test13ProjectManagerLoader() {
     assertEquals(5, ProjectManagerSchema.getTableNames().size());
   }
 
-  @Test
-  @Disabled
-  // todo delete
-  void test14DCATLoader() {
-    Schema DCATSchema = database.createSchema(DCAT);
-    DataModels.Profile.DCAT.getImportTask(DCATSchema, true).run();
-    assertEquals(23, DCATSchema.getTableNames().size());
-  }
-
   @Test
   void test15DirectoryStagingLoader() {
     Schema directoryStaging = database.createSchema(DIRECTORY_STAGING);
     DataModels.Regular.BIOBANK_DIRECTORY_STAGING.getImportTask(directoryStaging, false).run();
     assertEquals(8, directoryStaging.getTableNames().size());
   }
 
+  //  @Test
+  //  void test17FAIRDataPointLoader() throws Exception {
```

Comment:
we don't dare to delete this yet?

Reply:
I'll keep it in for now. I think we will delete them in the next PR since we aren't using the FDP profile anymore.